src/pages/Home.js

```diff
@@ -2,43 +2,40 @@ import styled from "styled-components";
 import { Link } from "react-router-dom";
 import Header from "../components/Header";
 import Footer from "../components/Footer";
+import { useContext } from "react";
+import { UserContext } from "../UserContext";
+
+export default function Home() {
+  const { config, setConfig } = useContext(UserContext);
 
-export default function Home({
-  config,
-  setConfig,
-  configForSave,
-  setConfigForSave,
-}) {
   function handleClickSectional() {
     setConfig({ ...config, system: "Sectionaltor" });
```

Comment:
```suggestion
    setConfig((prevConfigState) => ({ ...prevConfigState, system: "Sectionaltor" }));
```
Nothing wrong with how you're updating your state, but you could use the updater function here. That way, you are no longer dependent on the `config` state and don't need to destructure it off your useContext.

Reply:
I have not understood this... I believe that it is correct, because the prevConfig should not be set at this point, but only after verification by checkInput on the Measure page.